### DISCUSSION TITLE
Vercel support

### DISCUSSION THREAD

**Discussion by k-dahl** [vercel/community#3204] [Help]
### Problem Description

Sorry if this is not the appropriate place for this, but I've exhausted any other options.

I upgraded my account to Pro today as my trial ran out, but each time I try and create a support case it still says Hobby.

There seemed to be some sort of issue with the upgrade, as the first time I tried it said there was a problem with one of the 'team members', and it only billed us for one seat... i did the upgrade again, and it billed us for the correct number of seats and shows 'Pro' in the UI - but it does not look like any Pro features are actually enabled for us, i.e., - deployment protection of preview branches does not work.

Is there any other way to contact a human at Vercel for support? When I put in an e-mail support case it's saying the response time right now is going to be 2-4 days.

**Comment by amyegan** [comment]
Sorry for the delay. I just noticed your post in the CLI repo, so I moved it over here to our main support discussion board.

I found your support case and am glad to see that your Pro features are working correctly now. Please reach out in this [community support repo](https://github.com/orgs/vercel/discussions) or [submit a new case](https://vercel.com/help#issues) to the Customer Success Team if you need any help in the future 🙂
